src/lib/rmt/ssl/certificate_generator.rb

```diff
@@ -58,6 +58,23 @@ def ca_present?
     false
   end
 
+  def ca_encrypted?
+    !valid_password?(' ') # check with emtpy password. password has one char otherwise command requires input
```

Comment:
Since this is a little hack, can you document it in the code that we know after one year why this was there

Reply:
nice idea, but where exactly it has to be?

Reply:
Above, below, next to it but you already added a comment so its fine